packages/sdk/src/integration/client/routes.ts

```diff
@@ -125,9 +125,14 @@ type UserResponse<TIntegration extends BaseIntegration> = {
   >
 }
 
-export type CreateUser<TIntegration extends BaseIntegration> = (x: {
-  tags: ToTags<keyof TIntegration['user']['tags'], PrefixConfig<TIntegration>>
-}) => Promise<UserResponse<TIntegration>>
+export type CreateUser<TIntegration extends BaseIntegration> = (
+  x: Merge<
```

Comment:
this type basically means:
- all the properties of the actual `createUser` method in the client
- override the tags so they are contextual to the integration you're developing